scripts/exact-example.jl

```diff
@@ -0,0 +1,237 @@
+    u, t = convert_units(-10, M)
+    address = init_address(M)
+    ham = HubbardMom1D(address; u, t, dispersion=continuum_dispersion)
+    eigvals(Matrix(ham))[1]
+end
+
+# For the second, we will use
+# [`Arpack.jl`](https://github.com/JuliaLinearAlgebra/Arpack.jl).  Note that the `which=:SR`
+# argument tells Arpack to find the eigenvalues with the smallest real part first. `nev`
+# sets the number of eigenvalues to find. This is best used with moderately sized
+# Hamiltonians, where enough memory is available to build the matrix.
+
+# The transcorrelated Hamiltonian is non-Hermitian, so its eigenvalues will be complex.
+# You should make sure their imaginary part is zero. If it's not, try increasing the
+# `cutoff` parameter.
+
+trcorr_energies = map(5:2:20) do M
+    u, t = convert_units(-10, M)
+    address = init_address(M)
+    ham = Transcorrelated1D(address; v=u, t)
+    real.(eigs(sparse(ham); which=:SR, nev=1)[1][1])
+end
+
+# For the last method, we use the matrix-free method from
+# [KrylovKit.jl](https://github.com/Jutho/KrylovKit.jl/). Note that KrylovKit has no way of
+# knowing the Hamiltonian in question is symmetric. We mitigate that by passing it the
+# `issymmetric=true` keyword argument. It is only recommended to use this method when the
+# matrix is too large to store in memory as it tends to be much slower than the others.
+# Another good option is to use KrylovKit on a sparse matrix. This can sometimes outperform
+# `eigs`.
+
+renorm_energies = map(5:2:20) do M
+    u, t = convert_units(-10, M; renormalize=true)
+    address = init_address(M)
+    ham = HubbardMom1D(address; u, t, dispersion=continuum_dispersion)
+    vec = DVec(address => 1.0)
+    eigsolve(ham, vec, 1, :SR; issymmetric=true)[1][1]
+end
+
+# Now that we have the energies, let's compare how close to the reference energy they get.
+
+p = scatter(
+    5:2:20, abs.(lattice_energies .- reference_energy); label="lattice",
+    title="Convergence", xscale=:log10, yscale=:log10, legend=:bottomleft,
+    xlabel=L"M", ylabel=L"|E_0 - E_{\mathrm{ref}}|",
+)
+scatter!(p, 5:2:20, abs.(trcorr_energies .- reference_energy); label="transcorrelated")
+scatter!(p, 5:2:20, abs.(renorm_energies .- reference_energy); label="renormalized")
+
+# We can see that in this case, both lattice-renormalized and transcorrelated Hamiltonians
+# give good results even with modest values of ``M``. Since the values converge as a power
+# law, it's also possible to extrapolate the energies with a fitting package like
+# [LsqFit.jl](https://github.com/JuliaNLSolvers/LsqFit.jl). Keep in mind, however, that for
+# larger systems with very low ``M``, the convergence may not follow a power law.
+
+# ## Speeding up the computations
+
+# As these matrices tend to get large quickly, memory is usually the bottleneck.
+
+# There are currently two methods implemented to reduce the matrix size, `ParitySymmetry`
+# and `TimeReversalSymmetry`. You should only use these where the relevant symmetries
```

Comment:
```suggestion
# There are currently two methods implemented to reduce the matrix size, [`ParitySymmetry`](@ref)
# and [`TimeReversalSymmetry`](@ref). You should only use these where the relevant symmetries
```
It would be good to link to the documentation here. Documenter should be able to pick up these links.